dynawo/sources/Modeler/Common/DYNRingBuffer.cpp

```diff
@@ -29,6 +29,14 @@ RingBuffer::RingBuffer(double maxDelay) : queue_(), maxDelay_(maxDelay) {}
 
 void
 RingBuffer::add(double time, double value) {
+  if (queue_.size() > 0 && doubleEquals(queue_.back().first, time)) {
+#if _DEBUG_
+    assert(doubleEquals(queue_.back().second, value));
+#endif
+    // ignore if we add multiple time the same value
```

Comment:
When would this happen?

Reply:
When we perform dump then load, the last value dumped corresponds to the same value that will be put into the buffer after init (in a normal run, the call that initialiazes the delays)